mkdocs.yml

```diff
@@ -3,30 +3,36 @@ site_url: "https://docs.autolabproject.com/"
 extra_css:
     - stylesheets/extra.css
 nav:
-    - Autolab:
-        - Getting Started: index.md
-        - Guide for Instructors: instructors.md
-        - Guide for Lab Authors: lab.md
-        - Features:
-            - Formatted Feedback: features/formatted-feedback.md
-            - Scoreboards: features/scoreboards.md
-            - Embedded Forms: features/embedded-forms.md
-            - Annotations: features/annotations.md
-            - MOSS Plagiarism Detection: features/moss.md
-            - Metrics: features/metrics.md
+    - Overview: index.md
+    - Installation:
+        - Installation Overview: installation/overview.md
+        - Docker Compose Install: installation/docker-compose.md
+        - Autolab:
+            - Ubuntu 18.04: installation/ubuntu.md
+            - Mac OSX 10.11+: installation/osx.md
+        - Tango: installation/tango.md
+        # - (Legacy) OneClick Install: installation/one-click.md
+    - Getting Started:
+        - Autolab Frontend:
+            - Guide for Instructors: instructors.md
+            - Guide for Lab Authors: lab.md
+            - Features:
+                - Formatted Feedback: features/formatted-feedback.md
+                - Scoreboards: features/scoreboards.md
+                - Embedded Forms: features/embedded-forms.md
+                - Annotations: features/annotations.md
+                - MOSS Plagiarism Detection: features/moss.md
+                - Metrics: features/metrics.md
+        - Tango Basics: tango.md
     - Autolab API:
         - Overview: api-overview.md
         - Reference: api-interface.md
         - Command Line Interface: command-line-interface.md
         - Managing Authorized Apps: api-managing-authorized-apps.md
     - Tango:
```

Comment:
nit: What are your thoughts on calling this "Tango Resources" instead? Since there's also some critical information about Tango in the other sections for installation and getting started, I'm wondering if it might be confusing to see a general Tango tab. But I also don't see it being a big issue, so your call here.

Reply:
I agree we definitely need more precise terms to describe the different sections for Tango but I'm just thinking whether the term resources would imply installation and reference resources too. I'll make a note and we can discuss this once the new documentation comes to place as a whole.